Code/Framework/AzToolsFramework/CMakeLists.txt

```diff
@@ -31,6 +31,7 @@ ly_add_target(
     BUILD_DEPENDENCIES
         PRIVATE
             3rdParty::SQLite
+            3rdParty::xxhash
```

Comment:
What's the definition of supported?  We use xxhash for all of the file hashing in AP.  The speeds are similar depending on what version of cityhash we have, both score a 10 for quality: https://github.com/Cyan4973/xxHash

Reply:
I am cool either way, just wanted to know if there was a single hash package we wanted to use

Reply:
For what its worth, [Prefabs use cityhash to generate persistent entity ids](https://github.com/o3de/o3de/blob/38261d08007a1ad14135cf790cb855ccca4d8595/Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Instance/InstanceEntityIdMapper.cpp#L174-L178). So, it might be a good idea to use the same here as well.

Reply:
Hm, I guess I can switch it to be consistent then.